contribs/github-bot/internal/utils/testing.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/xlab/treeprint"
+)
+
+func TestLastNodeStatus(t *testing.T, success bool, details treeprint.Tree) bool {
```

Comment:
This is unused, right?

Reply:
This is used in tests only.

Reply:
now I see; for some reason when I tested it the references were not popping up.